.circleci/config.yml

```diff
@@ -9,6 +9,9 @@ jobs:
             - image: circleci/node:8.12.0-browsers
             - image: circleci/postgres:9.6.5-alpine-ram
             - image: 0xorg/ganache-cli:6.0.0
+              environment:
+                SNAPSHOT_NAME: 0x_ganache_snapshot_gene
```

Comment:
When will we be able to remove this?

Reply:
When the monorepo PR merges, I will publish a new snapshot version to the regular name, and then I will remove this.